src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -147,50 +153,37 @@ public String getScript() {
         }
 
         String scriptPath = shf.getRemote();
-        List<String> args = new ArrayList<>();
-
-        if (os != OsType.DARWIN) { // JENKINS-25848
-            args.add("nohup");
-        }
         if (os == OsType.WINDOWS) { // JENKINS-40255
             scriptPath= scriptPath.replace("\\", "/"); // cygwin sh understands mixed path  (ie : "c:/jenkins/workspace/script.sh" )
         }
 
-        envVars.put(cookieVariable, "please-do-not-kill-me");
         // The temporary variable is to ensure JENKINS_SERVER_COOKIE=durable-… does not appear even in argv[], lest it be confused with the environment.
-        String cmd;
-        FilePath logFile = c.getLogFile(ws);
-        FilePath resultFile = c.getResultFile(ws);
-        FilePath controlDir = c.controlDir(ws);
-        if (capturingOutput) {
-            cmd = String.format("pid=$$; { while [ \\( -d /proc/$pid -o \\! -d /proc/$$ \\) -a -d '%s' -a \\! -f '%s' ]; do touch '%s'; sleep 3; done } & jsc=%s; %s=$jsc %s '%s' > '%s' 2> '%s'; echo $? > '%s.tmp'; mv '%s.tmp' '%s'; wait",
-                controlDir,
-                resultFile,
-                logFile,
-                cookieValue,
-                cookieVariable,
-                interpreter,
-                scriptPath,
-                c.getOutputFile(ws),
-                logFile,
-                resultFile, resultFile, resultFile);
+        envVars.put(cookieVariable, "please-do-not-kill-me");
+
+        String arch = ws.act(new getArchitecture());
+        List<String> launcherCmd = null;
+        String launcher_binary = LAUNCHER_PREFIX + os.toString() + arch;
```

Comment:
Java convention is camelCase
```suggestion
        String launcherBinary = LAUNCHER_PREFIX + os.toString() + arch;
```